assets/src/globals.js

```diff
@@ -1,7 +1,9 @@
 import { createMuiTheme } from '@material-ui/core/styles'
 import defaultPalette from './defaultPalette'
 
-const mylaGlobals = JSON.parse(document.getElementById('myla_globals').textContent)
+// The fallback to an empty object is needed to not break tests using siteTheme.
+const mylaGlobalsEl = document.getElementById('myla_globals')
+const mylaGlobals = mylaGlobalsEl !== null ? JSON.parse(mylaGlobalsEl.textContent) : {}
```

Comment:
@ssciolla I think you can replace these two lines with:

```js
const mylaGlobals = document.getElementById('myla_globals') || {}
```
I think this is called JavasScript short circuiting: https://codeburst.io/javascript-what-is-short-circuit-evaluation-ff22b2f5608c

Reply:
Hm, I hadn't really ever tried this. Thanks for pointing it out. We still need to parse the JSON and use `.textContent`. Because the null check needs to happen before we use `.textContent`, I'm not sure how to really get this on one line. Thoughts?

Reply:
Good point, your implementation is totally fine. I think I would have done something like:

```js
const mylaGlobals = document.getElementById('myla_globals')
   ? JSON.parse(document.getElementById('myla_globals').textContent)
   : {}
```

Reply:
very similar, at the end of the day, I'm happy with either

Reply:
Thanks! I don't love the repetition of `document.getElementByID`, but I could change the beginning of the second line of my version to just `mylaGlobalsEl`, basically like you suggested. I'll try that.

Reply:
👍 otherwise LGTM!